app/inline-scripts/tracking-internal.js

```diff
@@ -3,10 +3,14 @@
 		return;
 	}
 
-	const shoeboxTrackingData = JSON.parse(document.querySelector('#shoebox-trackingData').innerHTML);
+	const trackingData = document.querySelector('#shoebox-trackingData');
```

Comment:
IMO this should be named `trackingDataShoebox` and the one below just `trackingData`